Framework/Tools/Emulator/Ssl.cs

```diff
@@ -608,9 +650,13 @@ public int DataAvailable( int socket )
 
             if (!GetSslData(socket, out ssd)) return (int)SocketError.SocketError;
 
-            if(!_socketsDriver.GetSocket(socket, out sock)) return (int)SocketError.SocketError;
+            if (!_socketsDriver.GetSocket(socket, out sock)) return (int)SocketError.SocketError;

```

Comment:
Please split this line, never put the return on the same line (While there's a ton of legacy code in NETMF some of which has this practice we are trying to clean it up so let's not add any new code following that style)